config/default.js

```diff
@@ -1,5 +1,5 @@
-import getPackageDetails from '@zap/utils/getPackageDetails'
-import isStableVersion from '@zap/utils/isStableVersion'
+import getPackageDetails from '../utils/getPackageDetails'
```

Comment:
What's the reason for switching this to relative imports?

Reply:
Wasn't able to access utils from e2e otherwise for some reason

Reply:
Seems that Testcafe doesn't use babel settings from the project, so our `zap` alias is not picked up. Their current  recommendation is for you to precompile your tests if you want to use custom babel settings.